snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```diff
@@ -192,6 +229,41 @@ metrics:
         table: entPhysicalTable
         tag: entity_name
 
+  - MIB: CISCO-STACKWISE-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.500.1.2.1
+      name: cswSwitchInfoTable
+    symbols:
+      - name: cswSwitchInfo
+        constant_value_one: true
+    metric_tags:
+      - tag: mac_addr
+        column:
+          OID: 1.3.6.1.4.1.9.9.500.1.2.1.1.7
+          name: cswSwitchMacAddress
+      - MIB: ENTITY-MIB
+        column:
+          OID: 1.3.6.1.2.1.47.1.1.1.1.6
+          name: entPhysicalIndex
+        table: entPhysicalTable
+        tag: entity_physical_index
```

Comment:
- this is the wrong OID for [entPhysicalIndex](https://www.circitor.fr/Mibs/Html/E/ENTITY-MIB.php#entPhysicalIndex)
- `entPhysicalIndex` is not accessible that's why we use `entPhysicalName` to index this table

Reply:
Still wondering where I took this OID... Thanks !